[2025-01-29T00:04:33.218+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T00:04:33.229+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2025-01-29T00:04:32.762764+00:00 [queued]>
[2025-01-29T00:04:33.234+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2025-01-29T00:04:32.762764+00:00 [queued]>
[2025-01-29T00:04:33.236+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T00:04:33.246+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_table> on 2025-01-29 00:04:32.762764+00:00
[2025-01-29T00:04:33.252+0000] {standard_task_runner.py:72} INFO - Started process 1376 to run task
[2025-01-29T00:04:33.255+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'executing_sql_pipeline', 'create_table', 'manual__2025-01-29T00:04:32.762764+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/executing_sql_pipeline.py', '--cfg-path', '/tmp/tmpytfu97w7']
[2025-01-29T00:04:33.257+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask create_table
[2025-01-29T00:04:33.292+0000] {task_command.py:467} INFO - Running <TaskInstance: executing_sql_pipeline.create_table manual__2025-01-29T00:04:32.762764+00:00 [running]> on host 6b2a71fc0c6d
[2025-01-29T00:04:33.358+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='executing_sql_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T00:04:32.762764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T00:04:32.762764+00:00'
[2025-01-29T00:04:33.360+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T00:04:33.368+0000] {sql.py:278} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                    id SERIAL PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    city VARCHAR(50),
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        
[2025-01-29T00:04:33.375+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-01-29T00:04:33.381+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-01-29T00:04:33.384+0000] {sql.py:553} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                    id SERIAL PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    city VARCHAR(50),
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        , parameters: None
[2025-01-29T00:04:33.405+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T00:04:33.406+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_sql_pipeline, task_id=create_table, run_id=manual__2025-01-29T00:04:32.762764+00:00, execution_date=20250129T000432, start_date=20250129T000433, end_date=20250129T000433
[2025-01-29T00:04:33.428+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T00:04:33.445+0000] {taskinstance.py:3895} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-01-29T00:04:33.453+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
